### PR TITLE
refactor(tests): implementation of helper files across all test types

### PR DESCRIPTION
**Pull request by moriahmaney**

### CODE REVIEW

File: test/journeys/specs/oneOnOne/global/messaging.js [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+      assert.equal(eventCreated.detail.data.actorName, mccoy.displayName);
+
+      assert.isDefined(eventUnread, `has an unread message event`);
+      assert.containsAllKeys(eventUnread.detail, [`resource`, `event`, `data`]);
+      assert.containsAllKeys(eventUnread.detail.data, [`actorId`, `actorName`, `id`, `title`, `type`, `created`, `lastActivity`]);
+      assert.equal(eventCreated.detail.actorId, constructHydraId(`PEOPLE`, mccoy.id));
+      assert.equal(eventCreated.detail.data.actorName, mccoy.displayName);
+    });
+
+    it(`sends and deletes message`);
```
Comment (adamweeks) [inline test/journeys/specs/oneOnOne/global/messaging.js]: I think you forgot something 🤣

File: test/journeys/specs/oneOnOne/global/messaging.js [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+      it(`sends message with png attachment`, () => {
+        sendFileTest(browserLocal, browserRemote, mccoy, `png-sample.png`);
+      });
+
+      it(`sends message with mp3 attachment`, () => {
+        sendFileTest(browserLocal, browserRemote, mccoy, `mp3-sample.mp3`);
+      });
+    });
+
+    it(`sends and flags message`);
```
Comment (adamweeks) [inline test/journeys/specs/oneOnOne/global/messaging.js]: Forgot something here too.

File: test/journeys/specs/oneOnOne/global/messaging.js [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+
+    describe(`markdown messages`, () => {
+      beforeEach(`wait until widget is loaded`, () => {
```
Comment (adamweeks) [inline test/journeys/specs/oneOnOne/global/messaging.js]: I'm not sure this entire `beforeEach` is needed any more. Could you verify?

File: test/journeys/lib/test-helpers/basic.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+export const elements = {
```
Comment (adamweeks) [inline test/journeys/lib/test-helpers/basic.js]: Could we make this file `test-helpers/space-widget/main.js` and move `meet.js` and `messaging.js` into this folder?

Comment (moriahmaney) [inline test/journeys/lib/test-helpers/basic.js]: do you want the containing folder named space-widget even though these are used across the different widget types (oneOnOne, recents, etc)?

Comment (adamweeks) [inline test/journeys/lib/test-helpers/basic.js]: I don't think we are using these in the recents widget. `oneOnOne` and `space` are using the same `space-widget`, just in different modes.

Comment (moriahmaney) [inline test/journeys/lib/test-helpers/basic.js]: oooh you're right we aren't using it anywhere else. thanks for clarifying, i'll make these changes now!

**Comment by adamweeks** [comment]
img 
LGTM

**Pull request merged**